test/spec/modules/33acrossAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,2571 @@
+    )
+  );
+}
+
+function getLocalAssert() {
+  // Derived from ReportDefnition.cue -> AnalyticsReport
+  function isAnalyticsReport(report) {
+    assert.hasAllKeys(report, ['analyticsVersion', 'pid', 'src', 'pbjsVersion', 'auctions']);
+    assert.equal(report.analyticsVersion, '1.0.0');
+    assert.isString(report.pid);
+    assert.isString(report.src);
+    assert.isString(report.pbjsVersion);
+    assert.isArray(report.auctions);
+    assert.isAbove(report.auctions.length, 0);
+    report.auctions.forEach(isAuction);
+  }
+  function isAuction(auction) {
+    assert.hasAllKeys(auction, ['adUnits', 'auctionId', 'userIds']);
+    assert.isArray(auction.adUnits);
+    assert.isString(auction.auctionId);
+    assert.isArray(auction.userIds);
+    auction.adUnits.forEach(isAdUnit);
+  }
+  function isAdUnit(adUnit) {
+    assert.hasAllKeys(adUnit, ['transactionId', 'adUnitCode', 'slotId', 'mediaTypes', 'sizes', 'bids']);
+    assert.isString(adUnit.transactionId);
+    assert.isString(adUnit.adUnitCode);
+    assert.isString(adUnit.slotId);
+    assert.isArray(adUnit.mediaTypes);
+    assert.isArray(adUnit.sizes);
+    assert.isArray(adUnit.bids);
+    adUnit.mediaTypes.forEach(isMediaType);
+    adUnit.sizes.forEach(isSizeString);
+    adUnit.bids.forEach(isBid);
+  }
+  function isBid(bid) {
+    assert.containsAllKeys(bid, ['bidder', 'bidId', 'source', 'status', 'hasWon']);
+    if ('bidResponse' in bid) {
+      isBidResponse(bid.bidResponse);
+    }
+    assert.isString(bid.bidder);
+    assert.isString(bid.bidId);
+    assert.isString(bid.source);
+    assert.oneOf(bid.status, [...Object.values(BID_STATUS), 'pending']);
+    assert.oneOf(bid.hasWon, [0, 1]);
+  }
+  function isBidResponse(bidResponse) {
+    assert.containsAllKeys(bidResponse, ['mediaType', 'size', 'cur', 'cpm', 'cpmFloor']);
+    if ('cpmOrig' in bidResponse) {
+      assert.isNumber(bidResponse.cpmOrig);
+    }
+    isMediaType(bidResponse.mediaType);
+    isSizeString(bidResponse.size);
+    assert.isString(bidResponse.cur);
+    assert.isNumber(bidResponse.cpm);
+    assert.isNumber(bidResponse.cpmFloor);
+  }
+  function isMediaType(mediaType) {
+    assert.oneOf(mediaType, ['banner', 'video', 'native']);
+  }
+  function isSizeString(size) {
+    assert.match(size, /[0-9]+x[0-9]+/);
+  }
+
+  function calledOnceWithStringJsonEquivalent(sinonSpy, ...args) {
+    sinon.assert.calledOnce(sinonSpy);
+    args.forEach((arg, i) => {
+      const stubCallArgs = sinonSpy.firstCall.args[i]
+
+      if (typeof arg === 'object') {
+        assert.deepEqual(JSON.parse(stubCallArgs), arg);
+      } else {
+        assert.strictEqual(stubCallArgs, arg);
+      }
+    });
+  }
+
+  sinon.assert.expose(assert, { prefix: '' });
+  return {
+    ...assert,
+    calledOnceWithStringJsonEquivalent,
+    isAnalyticsReport,
+    isAuction,
+    isAdUnit,
+    isBid,
+    isBidResponse,
+    isMediaType,
+    isSizeString
+  }
+};
+
+function getStandardAnalyticsReport() {
```

Comment:
A couple of naming notes:

- `get*` should be reserved for getters / query methods. This function is building something a new object from scratch, so I would recommend prefixing with `make*`, `create*`, or `build*` instead.
- There isn't anything standard about this analytics report; rather, looking at how the test cases make assertions on this report, its most relevant characteristic is that there are three auctions.

Hence, I would rename this function to something like `createAnalyticsReportWithThreeAuctions`.